Modules/Bibliographic/classes/Field/class.ilTable1.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+use ILIAS\UI\URLBuilder;
+
+/**
+ * Class ilTable
+ *
+ */
+class ilTable1
+{
+    use \ILIAS\Modules\OrgUnit\ARHelper\DIC;
+    protected array $filter = [];
+    protected ILIAS\UI\Implementation\Component\Table\Data $table;
+    protected \ilBiblAdminFactoryFacadeInterface $facade;
+
+
+    /**
+     * ilTable constructor.
+     */
+
+    public function __construct(?object $a_parent_obj, ilBiblAdminFactoryFacadeInterface $facade)
+    {
+        $this->facade = $facade;
+        $this->parent_obj = $a_parent_obj;
+        global $DIC;
+        $f = $DIC['ui.factory'];
+        $r = $DIC['ui.renderer'];
+        $df = new \ILIAS\Data\Factory();
+        $refinery = $DIC['refinery'];
+        $here_uri = $df->uri($DIC->http()->request()->getUri()->__toString());
+
+        $columns = [
+            //'position' => $f->table()->column()->text($this->lng()->txt('order')),
+            'identifier' => $f->table()->column()->text($this->lng()->txt('identifier')),
+            'data_type' => $f->table()->column()->text($this->lng()->txt('translation')),
+            'is_standard_field' => $f->table()->column()->text($this->lng()->txt('standard'))
+        ];
+
+        $url_builder = new URLBuilder($here_uri);
+
+        //these are the query parameters this instance is controlling
+        $query_params_namespace = ['bibliographic', 'admin'];
+        list($url_builder, $id_token, $action_token) = $url_builder->acquireParameters(
+            $query_params_namespace,
+            "row_id",
+            "table_action"
+        );
+
+        $actions = [
+            'translate' => $f->table()->action()->single(
+                'Translate',
```

Comment:
Übersetzungen müssen immer mit `$this->language->txt(...)` übersetzt werden.

dazu muss `$this->language` im konstruktor ebenfalls deklariert werden 

```
$this->language = $DIC->language();
```

und das property deklariert werden:

```
protected ilLanguage $language
```

Reply:
Die Aktion kann übers GUI bisher nicht aufgerufen werden.